package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "bg",
```

That's not really the name of out app, is it?
```suggestion
  "name": "bodgingbear.dev",
```